b2share/modules/records/errors.py

```diff
@@ -36,3 +41,16 @@ class AlteredRecordError(B2ShareRecordsError):
     """Raise when a record update changes what is considered
        immutable record data."""
 
+
+def register_error_handlers(app):
+    @app.errorhandler(ValidationError)
+    def handle_validation_error(err):
+        field = '/'.join(err.path)
+        if err.validator == 'required' or err.validator == 'additionalProperties':
+            try:
+                field = err.message.split('\'')[1]
+            except Exception:
```

Comment:
Catching silently all exceptions might hide some nasty bugs. Why not just catch `IndexError` in case the string is not as expected and the split returns an array with len < 2?